javascript/javascript1/week2/event-application.js

```diff
@@ -0,0 +1,35 @@
+getEventWeekday = (daysToEvent) => {
+  const d = new Date();
```

Comment:
I'll suggest you use a descriptive variable name here as well for the date object.
You can use something like currentDate or something even better

Reply:
committed changes